### DISCUSSION TITLE
What is the name of the feature that allows more complex "popups" on mouse hovers?

### DISCUSSION THREAD

**Discussion by markpiller** [squidfunk/mkdocs-material#6890] [Q&A] [answered]
I am talking about the highlighted area here: Can someone point me to the place in the docs where it is described how to configure it?

![question-illustration](https://github.com/squidfunk/mkdocs-material/assets/120929/ca95a334-1b4e-478a-b4a5-92930c17451d)

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @markpiller,
those are Instant Previews, the newest experimental addition to the Insiders features:
- https://github.com/squidfunk/mkdocs-material/issues/6704
- https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#instant-previews

This feature is limited to sponsors: https://squidfunk.github.io/mkdocs-material/insiders/
